src/main/java/hudson/remoting/Util.java

```diff
@@ -60,8 +60,11 @@ static File makeResource(String name, byte[] image) throws IOException {
         resource.getParentFile().mkdirs();
 
         FileOutputStream fos = new FileOutputStream(resource);
-        fos.write(image);
-        fos.close();
+        try {
+            fos.write(image);
+        } finally {
```

Comment:
Sure. But the PR is about Java 6. Upgrade to Java 7 is planned to remoting 3. See #103